documentation/src/main/asciidoc/ch12.asciidoc

```diff
@@ -40,6 +40,8 @@ getter methods (optionally, see below)
 annotations themselves
 * definition of dynamic default group sequence with @GroupSequenceProvider is valid
 * annotation parameter values are meaningful and valid
+* method parameter constraints in inheritance hierarchies are applied correctly, except some cases of parallel implementation.
```

Comment:
Is the "except some cases of parallel implementation" still true with the new implementation?